index.bs

```diff
@@ -2231,6 +2237,139 @@ and makes it clear when the state described by the flags is reset.
+One downside to this approach
+is that the difference in behavior is not [feature detectable](#feature-detect).
+On the other hand,
+if something changes in the future
+that enables all implementations to converge on the same behavior,
+sites will not need to be updated
+to take advantage of this.
+
+Note: Authors may assume the behavior of the dominant implemnentation is correct
+and any other behavior is buggy,
+which may further entrench the dominant implementaiton.
+
+<div class=example id=unknown1>
+
+The 'backdrop-filter' property is known to have a number of visible
+behavior differences between implementations.
+At the present time,
+those working on it believe
+the benefits of sharing an API
+outweigh the interoperability costs of the differences.
+
+</div>
+
+If the behavior differences are serious enough,
+it may be preferable to specify different APIs.
+
+In this scenario,
+each implementor goes with the alternative
+that they are able to implement.
+If implementations are eventually able to converge on the same behavior,
+both APIs may need to be supported indefinitely,
+which can increase implementation maintenance costs.
+
+Exposing two different APIs
+that are intended to address the same use cases
+can be confusing to authors.
+Authors can use [feature detection](#feature-detect)
+to write different code for different browsers.
+
+Note: There is a risk that authors may assume
+the API variant supported by the dominant implementation
+is the "correct" one,
+or they may simply be unaware of the API variants supported
+in implementations they do not use regularly,
+which may further entrench the dominant implementaiton.
+
+If you do go this route,
+try to reduce the cost to authors
+by minimizing the differences between the APIs.
+It may be possible to design things
+such that the non-standard part
+is a small, isolated component
+of a larger, standardized framework.
+
+<div class=example id=eme-and-payment-request>
+
+[[ENCRYPTED-MEDIA]] and [[PAYMENT-REQUEST]] are both examples of specifications
+which minimize API differences by isolating a non-standard component
+([=Content Decryption Modules=] and [=payment methods=], respectively)
+from the rest of the (shared) API surface.
+
+</div>
+
+It is also possible to specify an API that not all browsers will implement.
+Authors may use feature detection to only use the API it is implemented,
+and users may benefit where the feature is supported and used.
+This is always a problematic outcome,
+though how severe the problem is may depend somewhat on
+the number and market share of the implementations willing or unwilling
+to implement the feature.
+
+Note: This is worst when there is only one willing implementor.
+Some standards venues have rules
+explicitly disallowing the standardization of features
+which only have a single interested implementor.
+Even if you are not operating in such a group,
+standardization of single-implementation features is strongly discouraged.
```

Comment:
It might be worth elaborating on some of the risks here. Such as bugs getting entrenched, the specification and tests highly likely being of poor quality, etc.

Reply:
Yeah, I think it would be good to elaborate a bit here. But I think it's okay to land as-is and elaborate in subsequent PRs.